### DISCUSSION TITLE
Ergo: block effort

### DISCUSSION THREAD

**Discussion by Censseo** [coinfoundry/miningcore#1322] [Q&A]
Hello, I'm trying to understand what to do with the block effort, because I don't really understand what the value sent by the api meaning. 
I looked into the code and if I understand well it's the SUM(difficulty/network_difficulty) from shares but the value sent cannot be true because I got something like 0.0033236580321025786, what I am supposed to do with that?

**Comment by totom9000** [comment]
Take value from effort  in api and multiply with 100  to get % of effort.
0.0033236= 0.33% effort

**Reply by Censseo** [reply]
this is what I don't understand, this is not the real effort, we have mined for 2 days at more than 10gh/s to get the block, it cannot be 0.33% effort

**Reply by totom9000** [reply]
Not shure wath coin  you do but might be share multiplyer  is wrong on coin 
If its wrong  it wil think your hashrate  is difreng than it is. 

But is it like this on all or Just this coin
Wuld be Nice to know witch to try  replicate foult aswell.

**Reply by Censseo** [reply]
okay, thks for the answer, it can be a good guess. Do u know if it's possible to change it in config file or is it hardcoded somehow? I would like to do some tests on testnet
I only have ergo right so I cannot say if it is the same on other algo

**Reply by MiningCryptoLive** [reply]
I can't find effort in the api. Was it removed on the latest build?

**Reply by oliverw** [reply]
No it was not.

**Comment by MiningCryptoLive** [comment]
How to add it so it is in here. Config used to have poolEffort at the end
with total blocks

On Sun, Jul 24, 2022, 2:42 PM Oliver Weichhold ***@***.***>
wrote:

> No it was not.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/oliverw/miningcore/discussions/1322#discussioncomment-3217035>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIAKRS3KMQRX3FRRE5U5HLVVWFATANCNFSM54L4THXA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
![Screenshot_20220724-145556.png](https://user-images.githubusercontent.com/1049926/180662054-2eb09879-6020-4113-8776-20d90b95c364.png)

**Comment by ei8ht187** [comment]
Since MC v70 (?), latest 71, we have the problem that the blockeffort for ERGO is always 1%, max 2%

**Reply by totom9000** [reply]
Do you  math it out ? Seem your always 100-200% effort 
Need take blockeffort number and x100

**Reply by oliverw** [reply]
> Do you math it out ? Seem your always 100-200% effort Need take blockeffort number and x100

I'm afraid that's not what he meant. He was referring to the displayed value that is already converted to percent. I think the problem is related to the difficulty changes I made to Ergo in the last  update.

**Reply by totom9000** [reply]
Wil take a look and see if we can relese  new deamon and MC update on this

**Reply by oliverw** [reply]
> Wil take a look and see if we can relese new deamon and MC update on this

Got any new info on this?

**Comment by btclinux** [comment]
Anybody fix this? Have the same issue

**Reply by xmrstickers** [reply]
issue still persists

**Comment by jvxis** [comment]
We found a block, and this is the effort.:  "effort": 0.0007439966460211121
![image](https://user-images.githubusercontent.com/108929149/189863786-66d368c3-c3aa-479f-b5c0-e94063bcfc0c.png)
Multiplying by 100, and considering that the value will be displayed as a %, the block effort is 7.4%.  What I think is ok, considering the pool hash and time to find it.
What I can't understand is why Miningcore UI is showing a block effort 1% instead of 7.4%. The value is so different that it seems that it was hard coded.
![image](https://user-images.githubusercontent.com/108929149/189865177-1df20a7d-a542-441a-a10e-5275e854353a.png)

**Comment by oliverw** [comment]
Can you guys try with the latest `dev` branch commit?

**Reply by totom9000** [reply]
Offered  you this fix yesterday  oliverw  you didnt want it.

**Reply by oliverw** [reply]
> Offered you this fix yesterday oliverw you didnt want it.

Could you stop spamming the issue log with that shit? You wanted something in exchange for it and offered not much in return.
